146866961 align map popup with pin

OlenaCh opened the pull request.

tansaku commented:
many thanks @OlenaCh ! :-)

tansaku commented:
btw, would be wonderful if we could conform to the PR elements we describe in the contributing file so that's a link to the tracker ticket in the description and a commit that includes the correct syntax to close the relevant tracker ticket?

https://github.com/AgileVentures/LocalSupport/blob/develop/CONTRIBUTING.md#pull-requests

Many thanks in advance :-)

OlenaCh commented:
@tansaku oh, sorry, I totally forgot about PT links at the top of PR requests. I've changed the PR description - added relevant info and also changed status of the ticket.
